website/docs/docs/collaborate/dbt-explorer-faqs.md

```diff
@@ -171,3 +171,11 @@ Yes, users with read-only access can use the dbt Explorer. Specific feature avai
 The ability to embed and share views is being evaluated as a potential future capability. 
 
 </Expandable>
+
+<Expandable alt_header="Can you access dbt Explorer from other features and products inside dbt Cloud?" >
```

Comment:
The phrasing may raise other questions like "What's the difference between a feature and a product?" 
```suggestion
<Expandable alt_header=" Is dbt Explorer accessible from other areas inside dbt Cloud?" >
```